argo_qa/schema.xml

```diff
@@ -68,131 +44,90 @@
-    <field name="date"        type="date"    indexed="true" stored="false" multiValued="true"/>
+    <!-- English text (_te...) -->
+    <dynamicField name="*_tei"    type="text_en" stored="false" indexed="true"  multiValued="false"/>
+    <dynamicField name="*_teim"   type="text_en" stored="false" indexed="true"  multiValued="true"/>
+    <dynamicField name="*_tes"    type="text_en" stored="true" indexed="false" multiValued="false"/>
+    <dynamicField name="*_tesm"   type="text_en" stored="true" indexed="false" multiValued="true"/>
+    <dynamicField name="*_tesi"   type="text_en" stored="true" indexed="true"  multiValued="false"/>
+    <dynamicField name="*_tesim"  type="text_en" stored="true" indexed="true"  multiValued="true"/>
+    <dynamicField name="*_teiv"   type="text_en" stored="false" indexed="true"  multiValued="false" termVectors="true" termPositions="true" termOffsets="true"/>
+    <dynamicField name="*_teimv"  type="text_en" stored="false" indexed="true"  multiValued="true"  termVectors="true" termPositions="true" termOffsets="true"/>
+    <dynamicField name="*_tesiv"  type="text_en" stored="true" indexed="true"  multiValued="false" termVectors="true" termPositions="true" termOffsets="true"/>
+    <dynamicField name="*_tesimv" type="text_en" stored="true" indexed="true"  multiValued="true"  termVectors="true" termPositions="true" termOffsets="true"/>
 
-    <dynamicField name="*_i"  type="tint"    indexed="true"  stored="true"/>
-    <dynamicField name="*_s"  type="string"  indexed="true"  stored="true" multiValued="true"/>
-    <dynamicField name="*_l"  type="tlong"   indexed="true"  stored="true"/>
-    <dynamicField name="*_b"  type="boolean" indexed="true"  stored="true"/>
-    <dynamicField name="*_f"  type="tfloat"  indexed="true"  stored="true"/>
-    <dynamicField name="*_d"  type="tdouble" indexed="true"  stored="true"/>
-    <dynamicField name="*_dt" type="date"    indexed="true"  stored="true" multiValued="true"/>
-    <dynamicField name="*_t"  type="text"    indexed="true"  stored="true" multiValued="true"/>
-    <dynamicField name="*_t_ws" type="text_ws" indexed="true" stored="false" multiValued="true"/>
-    <dynamicField name="*_t_unstem_search"  type="textNoStem" indexed="true"  stored="false" multiValued="true"/>
+    <dynamicField name="*_text_nostem_im"  type="textNoStem" indexed="true"  stored="false" multiValued="true"/>
 
     <dynamicField name="random*" type="rand" />
 
     <dynamicField name="*_sort"    type="alphaSort" indexed="true"  stored="false" multiValued="false" />
     <dynamicField name="*_dt_sort" type="tdate"     indexed="true"  stored="false" multiValued="false" />
-    <dynamicField name="*_facet"   type="string"    indexed="true"  stored="true" multiValued="true" />
-    <dynamicField name="*_display" type="string"    indexed="false" stored="true" multiValued="true" />
-    <!-- END argo customization artifacts -->
```

Comment:
the removed dynamicFields are not used in Argo and are even commented here as "artifacts" - I kept the ones we use